Chatkit.xcodeproj/project.pbxproj

```diff
@@ -1824,6 +1916,22 @@
 			};
 			name = Release;
 		};
+		73A9394723E1FB800025AF90 /* Debug */ = {
+			isa = XCBuildConfiguration;
+			buildSettings = {
+				CODE_SIGN_STYLE = Automatic;
```

Comment:
Does this target require to be code signed?

Reply:
I shouldn't have thought so.  All the build settings are just the Xcode defaults for an aggregate target.  

I can try and align the build settings with the other targets if you'd like?

Reply:
I would personally aim to have as few build settings overrides as possible and keep most of the configuration of the project level.

Code signing has been disabled for all the other target because we lost access to our Apple Developer Program and this was causing issues on Bitrise.

Reply:
I completely agree with the sentiment.  Ideally we'd use xcconfig's but thats not something we need to implement anytime soon. 

I have removed *all* of the build settings for `MakeInstanceLocatorFile`. 

Just reaslised that I named the target without thinking.  I'm happy to rename `Make InstanceLocator File` for consistency if you'd prefer?

Reply:
Just keep it as it is.